tutorials/yes-no-game/steps/step-5.md

```diff
@@ -143,6 +143,7 @@ Set `.input-received` to true to prevent further user input, and end the round i
     6. If it is not Game Over yet...
     7. ... end the round.
     8. [`Step`](https://docs.fragcolor.xyz/docs/shards/General/Step/) runs a wire inline.
+    9. `nil` ensures that any previous inputs (e.g. the Bool from `true > .input-received`) is not propagated into the wire
```

Comment:
maybe also hint the why, as in it would be a violation to change the input of an already used wire (the first use btw is the one defining it)

Reply:
An additional explanation has been added about wire input type definition and subsequent calls possibly violating that definition in commit [444677a](https://github.com/fragcolor-xyz/shards-examples/pull/57/commits/444677adc9784b5f9c7c8dff9c701920f1676dc6).